met-api/migrations/versions/ffac8f5b4288_add_reject_reason.py

```diff
@@ -0,0 +1,30 @@
+"""add reject reason
+
+Revision ID: ffac8f5b4288
+Revises: 46490dc02be5
+Create Date: 2022-12-07 12:11:33.172817
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'ffac8f5b4288'
+down_revision = '46490dc02be5'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('submission', sa.Column('has_personal_info', sa.Boolean(), nullable=True))
```

Comment:
if you want to write with less characters , i will go for has_PI , has_abuse etc..but sine you already done it , let it eb like this..
I am thinking of either just **rejected_reason** or **rejected_reason_text** than **rejected_reason_other** ;we might forget down the line whats the **other** for...just **rejected_reason** itself is pretty good i feel ;but no need to change if you dont feel so